marketdepthviewer/display.go

```diff
@@ -0,0 +1,169 @@
+package marketdepthviewer
+
+import (
+	"fmt"
+	"log"
+	"math/big"
+	"sort"
+	"time"
+
+	"github.com/gdamore/tcell/v2"
+)
+
+func initialiseScreen() error {
+	var e error
+	ts, e = tcell.NewScreen()
+	if e != nil {
+		log.Fatalln("Failed to create new tcell screen", e)
```

Comment:
`Fatal` will actually end the program, so returning after that is redundant. Better if we return instead of fatal